notebooks/instructlab-knowledge/utils/create_seed_dataset.py

```diff
@@ -159,21 +159,18 @@ def add_icls(qna_yaml: Dict[str, str], chunked_document: Dataset) -> Dataset:
             )
         )
     chunked_document_all_icl = safe_concatenate_datasets(chunked_document_all_icl)
-    chunked_document_all_icl = chunked_document_all_icl.map(
-        lambda x: {
-            "chunks": chunk_document(
```

Comment:
Looks like `chunks` still needs to be set as `x["document"]` in `chunked_document_all_icl`? 

The CI is failing on line 169 because there's not any "chunks"
Link: https://github.com/instructlab/examples/actions/runs/15569370134/job/43841310963?pr=29